examples/bond/scripts/deploy.js

```diff
@@ -0,0 +1,115 @@
+const {
+  executeTransaction, convert
+} = require('@algo-builder/algob');
+const { types } = require('@algo-builder/web');
+
+async function run (runtimeEnv, deployer) {
+  const masterAccount = deployer.accountsByName.get('master-account');
+  const storeManagerAccount = deployer.accountsByName.get('alice');
+  const creatorAccount = deployer.accountsByName.get('john');
+
+  const algoTxnParams = {
+    type: types.TransactionType.TransferAlgo,
+    sign: types.SignType.SecretKey,
+    fromAccount: masterAccount,
+    toAccountAddr: storeManagerAccount.addr,
+    amountMicroAlgos: 200e6,
+    payFlags: {}
+  };
```

Comment:
why do we pay 200 ALGO to the manager? It's more then he should need.

Reply:
ok, reducing it